features/plugin-activate.feature

```diff
@@ -148,3 +148,27 @@ Feature: Activate WordPress plugins
       Success:
       """
     And the return code should be 0
+
+  @require-wp-5.1
+  Scenario: Incompatible plugins cannot be activated
```

Comment:
This is the same as the `Activating a plugin that does not meet PHP minimum throws a warning` on line 108.

Can't we just amend it with the 

```
    And STDOUT should not contain:
      """
      Success:
      """
```

check?

Reply:
Huh! Yes, we can. I did not even check to see if an existing test was present, as I assumed the bug would otherwise not be possible... 😅

Reply:
I reused the existing test now. However, I tweaked the PHP version fetching a bit, as technically the `php` binary and the PHP that WP-CLI uses are not necessarily the same version.